bin/stylus

```diff
@@ -693,11 +700,7 @@ function writeFile(file, css) {
 */
 
 function writeSourcemap(file, sourcemap) {
-  // --out support
-  var out = basename(file, extname(file)) + '.css.map'
-    , path = dest
-      ? join(dest, out)
-      : join(dirname(file), out);
+  var path = createPath(file);
```

Comment:
There should be `<file>.css.map` not `<file>.css` so we should introduce an optional argument with extname to `createPath`.